mantidproject/mantid
# Indirect - ConvFit - Incorrect plotting in Miniplot

The output of running a convolution fit from the Indirect ConvFit interface was not correctly been used to update the miniplot in the interface, however, the main plot (and the plot obtained from clicking 'Plot Current Preview') was correct.

The fitting range (blue fit selectors) was also always defaulting back to it's original position when changing spectra, even if manually altered.

**Must be merged after #20642**

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > ConvFit.
2. Supply the given Sample (\*\_red) workspace in the 'Sample' field.
3. Supply the given Resolution (\*\_res) workspace in the 'Resolution' field.
4. Select 'One Lorentzian' in the 'Fit Type' drop-down menu.
5. Select 'Fit Flat' in the 'Background' drop-down menu.
6. Click 'Run'.
7. Navigate through each spectra and ensure a reasonable fit was made and matches the plot created by clicking 'Plot Current Preview'.
8. Click and drag the blue dotted lines in the mini-plot and change the 'Plot Spectrum' spinner, ensuring they remain in their new location.

**Test Files:**
[ConvFit-TestFiles.zip](https://github.com/mantidproject/mantid/files/1353254/ConvFit-TestFiles.zip)



Fixes #20697. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
